esphome/components/nrf52/gpio.py

```diff
@@ -0,0 +1,61 @@
+from esphome import pins
+
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.const import (
+    CONF_ID,
+    CONF_MODE,
+    CONF_INVERTED,
+    CONF_NUMBER,
+)
+from esphome.components.zephyr.const import (
+    zephyr_ns,
+)
+
+GPIOPin = zephyr_ns.class_("ZephyrGPIOPin", cg.InternalGPIOPin)
```

Comment:
Class names should always match here to cause less confusion
```suggestion
ZephyrGPIOPin = zephyr_ns.class_("ZephyrGPIOPin", cg.InternalGPIOPin)
```